src/Blocks/analysis_points.jl

```diff
@@ -183,15 +190,15 @@ function get_sensitivity_function(sys, ap_name::Symbol; loop_openings = nothing,
             if namespaced_ap_match(ap, ns, ap_name, nothing)
                 namespace[] = ns
                 apr[] = ap
-                (ap.out.u ~ ap.in.u + d), d
+                (ap_var(ap.out) ~ ap_var(ap.in) + d), d
```

Comment:
Should we also compare that their variable names equal?

Reply:
If there is only a single variable, I figured it was unambiguous no matter what it was called. If there were more than one variable, I threw an error instead. This happens already in the `ap_var` function. 

The reasoning for allowing any one variable is to make it more likely that it will work for user-defined components etc. without the user having built the components using our `RealInput/RealOutput` components.

Reply:
Making the matching based on name rather than type is a bit unfortunate in any case. An alternative approach would be to store the variable name in the analysis point, and default to `u` if none was provided. I'm not sure for about how the syntax for this would look though.